### PR TITLE
Print tags when printing schemas in the Cedar format

### PR DESCRIPTION
**Pull request by shaobo-he-aws**
## Checklist for requesting a review

The change in this PR is (choose one, and delete the other options):

- [ ] A breaking change requiring a major version bump to `cedar-policy` (e.g., changes to the signature of an existing API).
- [ ] A backwards-compatible change requiring a minor version bump to `cedar-policy` (e.g., addition of a new API).
- [x] A bug fix or other functionality change requiring a patch to `cedar-policy`.
- [ ] A change "invisible" to users (e.g., documentation, changes to "internal" crates like `cedar-policy-core`, `cedar-validator`, etc.)
- [ ] A change (breaking or otherwise) that only impacts unreleased or experimental code.

I confirm that this PR (choose one, and delete the other options):

- [x] Updates the "Unreleased" section of the CHANGELOG with a description of my change (required for major/minor version bumps).
- [ ] Does not update the CHANGELOG because my change does not significantly impact released code.

I confirm that [`cedar-spec`](https://github.com/cedar-policy/cedar-spec) (choose one, and delete the other options):

- [x] Does not require updates because my change does not impact the Cedar formal model or DRT infrastructure.
- [ ] Requires updates, and I have made / will make these updates myself. (Please include in your description a timeline or link to the relevant PR in `cedar-spec`, and how you have tested that your updates are correct.)
- [ ] Requires updates, but I do not plan to make them in the near future. (Make sure that your changes are hidden behind a feature flag to mark them as experimental.)
- [ ] I'm not sure how my change impacts `cedar-spec`. (Post your PR anyways, and we'll discuss in the comments.)

I confirm that [`docs.cedarpolicy.com`](https://docs.cedarpolicy.com/) (choose one, and delete the other options):

- [x] Does not require updates because my change does not impact the Cedar language specification.
- [ ] Requires updates, and I have made / will make these updates myself. (Please include in your description a timeline or link to the relevant PR in [`cedar-docs`](https://github.com/cedar-policy/cedar-docs). PRs should be targeted at a `staging-X.Y` branch, not `main`.)
- [ ] I'm not sure how my change impacts the documentation. (Post your PR anyways, and we'll discuss in the comments.)

### CODE REVIEW

**Review by cdisselkoen — APPROVED** [review]

**Comment by cdisselkoen** [comment]
We should backport this to 4.2 as well

**Review by john-h-kastner-aws — APPROVED** [review]
Do we have a test for printing tags in the JSON format?

File: cedar-policy-validator/src/json_schema.rs [hunk 1]
```diff
@@ -2546,7 +2545,21 @@ mod entity_tags {
...
+        let json = example_json_schema();
+        let json_schema = Fragment::from_json_value(json.clone()).expect("should be valid");
+        let serialized_json_schema = serde_json::to_value(json_schema).expect("should be valid");
+        assert_eq!(json, serialized_json_schema);
+    }
+
+    /// This schema taken directly from the RFC 82 text
+    #[test]
+    fn basic() {
+        let json = example_json_schema();
```
Comment (cdisselkoen) [inline cedar-policy-validator/src/json_schema.rs:2562]: This comment probably makes more sense to go on `example_json_schema()`

**Review by cdisselkoen — APPROVED** [review]

File: cedar-policy-validator/src/json_schema.rs (hunk 1 above)
Comment (shaobo-he-aws) [reply cedar-policy-validator/src/json_schema.rs]: Good catch. Fixed.

**Review by andrewmwells-amazon — APPROVED** [review]

**Pull request merged**